### ISSUE TITLE
`Flutter` installation failed

### ISSUE THREAD

**Issue by daiki1003** [subosito/flutter-action#179]
## ⛳️ Overview
- Flutter installation always ends with exit code 127.

## 🎯 Commits Info

`b1d33bca250325e536d154a6ae802a463c52bc57` works fine, but `c30358aafe5ae8e54d1ee4cbba0220fb4b311958` produces error.

## 📓 yaml file

```yaml
 - name: 'Setup flutter environment'
      uses: subosito/flutter-action@v2
      with:
        flutter-version: '3.0.4'
        channel: 'stable'
        cache: true
```

## 📸 Screenshot
![CleanShot 2022-07-26 at 16 32 44](https://user-images.githubusercontent.com/2792420/180949653-f8c1e4f1-30d0-4184-90b4-d39042b0bcab.png)

**Comment by dhimanrascon** [comment]
We've facing the same error

**Comment by KKimj** [comment]
We Should Downgrade to 2.4.0.

```yml

      - name: Setup Flutter SDK
        uses: subosito/flutter-action@v2.4.0
        with:
          flutter-version: ${{ matrix.flutter-version }}
          channel: ${{ matrix.flutter-channel }}
          cache: true
```

**Comment by daiki1003** [comment]
@KKimj Thanks, I will :)

**Comment by kouhon** [comment]
@KKimj Now it works. Thank you!

**Comment by wenkey-everest** [comment]
Facing the same error, downgrade to v2.4.0 solved it.

**Comment by alexeynobody** [comment]
2.5.0 also working good

```yaml
      - uses: subosito/flutter-action@v2.5.0
        with:
          channel: 'stable'
```

**Comment by dhimanrascon** [comment]
> ~We Should Downgrade to 2.4.0.~ edited) We Should use 2.5.0 version!!
> 
> ```yaml
>       - name: Setup Flutter SDK
>         uses: subosito/flutter-action@v2.5.0
>         with: 
>           channel: 'stable'
>           cache: true
> ```

Working.

**Comment by edufolly** [comment]
Facing the same issue with v2.6.0.

**Comment by hakunatomata2** [comment]
Same here, suddenly stopped working!
<img width="1304" alt="image" src="https://user-images.githubusercontent.com/90859024/180983943-893dc0aa-9321-4d65-aa7f-00b5d51b1b44.png">

**Comment by yurikoles** [comment]
Please stop posting useless comments, like `Me too`, this produces unwanted notifications for subscribers. Consider reacting with emoji to any post in the thread in order to subscribe to for updates

**Comment by subosito** [comment]
Just pushed a fix for this in `v2` and `v2.6.1`. Please retry and confirm if the issue is solved. Thanks!

**Comment by daiki1003** [comment]
@subosito 
Thanks a lot! :)
